boards/BMS/Inc/App/App_CurrentSense.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "App_ErrorCode.h"
+
+/**
+ * Convert the given ADC voltage to main current 1
+ * @param adc_voltage The ADC voltage to convert
+ * @param main_current_1 This will be set to main current 1, in amps
+ * @return EXIT_CODE_OUT_OF_RANGE if adc_voltage is negative
+ *         EXIT_CODE_INVALID_ARGS if main_current_1 is NULL
+ */
+ExitCode App_CurrentSense_ConvertToMainCurrent1(
```

Comment:
Originally I wanted `App_{Current|Voltage}Sense.{c|h}` to be in the IO layer because it's code for the current sensors. But I think it's better to leave as much as possible in the App layer so we can write unit tests for them.

Reply:
Also didn't put any of these functions in `BmsWorld` because it's tough to know where they belong right now. So just going to leave them as standalone helper functions for now.